pkg/ctrlz/topics/args.go

```diff
@@ -45,7 +47,8 @@ func (argsTopic) Activate(context fw.TopicContext) {
 		fw.RenderHTML(w, tmpl, os.Args)
 	})
 
-	_ = context.JSONRouter().StrictSlash(true).NewRoute().Methods("GET").Path("/").HandlerFunc(func(w http.ResponseWriter, req *http.Request) {
-		fw.RenderJSON(w, http.StatusOK, os.Args)
+	_ = context.JSONRouter().StrictSlash(true).NewRoute().Methods("PUT", "POST").Path("/reloadconfig").HandlerFunc(func(w http.ResponseWriter, req *http.Request) {
```

Comment:
This isn't the appropriate way to integrate with ControlZ behavior. This topic is there to return command-line flags. You should be adding a new Galley-specific topic for reloading config.

Reply:
I did this in an earlier version but I'm not sure it makes sense. The trigger here reloads configuration from files/dirs that are passed as command line argument

Also it's worth noting that this is not specific to Galley, its generic for any component that loads config in this manner. This is something that Pilot and Mixer are currently doing too even in production settings

Reply:
This topic is intended to show command-line options, I don't think it's the right place to put things that affect the behavior of the running component. I'd prefer having a distinct topic for this purpose.

Reply:
Done - moved into a 'commands' topic

Reply:
Should this change be undone then, and the one to args.html?